src/qmake/platforms/linux.pri

```diff
@@ -136,20 +139,33 @@ systemd_service.files = $$PWD/../../../linux/mozillavpn.service
 systemd_service.path = $${USRPATH}/lib/systemd/system
 INSTALLS += systemd_service
 
-manifestFirefox.files = $$PWD/../../../extension/manifests/linux/mozillavpn.json
-manifestFirefox.path = $${USRPATH}/lib/mozilla/native-messaging-hosts
+ORIG_MOZILLAVPN_JSON = $$PWD/../../../extension/manifests/linux/mozillavpn.json
+manifestFile.input = ORIG_MOZILLAVPN_JSON
+manifestFile.output = $$PWD/../../../linux/mozillavpn.json
```

Comment:
I think my only concern here is that this writes stuff into the source directory during the build, we should probably place the intermediate manifest file into `OBJECTS_DIR`, or make sure that `.gitignore` is updated accordingly.